### DISCUSSION TITLE
Welcome to GitactionBoard Discussions!

### DISCUSSION THREAD

**Discussion by sumanmaity1234** [otto-de/gitactionboard#74] [General]
👋 Welcome!
---
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, fell free to drop a comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by venkatesh-mb** [comment]
Hi, 
This is nice dashboard which i am using and it's giving lot of benefits to me.
I feel it's good to have the few features like 
1. Filtering of types of jobs and repositories.
2. Can we also add jobs which are in queue?
3. Can we also add some dashboard, like how many are failed, passed and in queue?

**Reply by sumanmaity1234** [reply]
Hi @venkatesh-mb,

- For point 1, could you please explain it little bit?
- For point 2, it'll be a nice addition to the dashboard. Do you want to give it a try and raise a PR. I will be happy to merge it :)
- For point 3, are you looking for some sort of [bar chart](https://en.wikipedia.org/wiki/Bar_chart) by job status?
